manifests/o/OpenTTD/OpenTTD/12.1/OpenTTD.OpenTTD.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: OpenTTD.OpenTTD
+PackageVersion: 12.1
+PackageLocale: en-US
+Publisher: OpenTTD
+PublisherUrl: https://www.openttd.org/
+PublisherSupportUrl: https://www.openttd.org/contact.html
+PrivacyUrl: https://www.openttd.org/policy.html
+PackageName: OpenTTD
+PackageUrl: https://www.openttd.org/
+License: GPL-2.0
+LicenseUrl: https://github.com/OpenTTD/OpenTTD/blob/master/COPYING.md
+ShortDescription: OpenTTD is a transport simulation game based upon the popular game Transport Tycoon Deluxe, written by Chris Sawyer. It attempts to mimic the original game as closely as possible while extending it with new features.
+Moniker: openttd
+Tags:
+- game
+- ttd
+- open source game
+- trnsport tycoon
+- tycoon
+ReleaseNotes: Feature Button to toggle showing advanced signal types (#9617), Change Don't show screenshot GUI in screenshots (#9674), Change Suppress panning in intro game, while user is interacting with the GUI (#9645), Change Draw rotor in cursor when dragging helicopters in depots (#9612), Fix Invalid memory access when loading a currency NewGRF (#9675), Fix
+ReleaseNotesUrl: https://cdn.openttd.org/openttd-releases/12.1/changelog.txt

```

More detailed release notes (with links) on github
```suggestion
ReleaseNotesUrl: https://github.com/OpenTTD/OpenTTD/releases/tag/12.1
```